From 7c29a1e8607b4950c5e36bc1cf1d5d4d021aa504 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 1 Aug 2019 19:44:00 +0300
Subject: [PATCH 629/741] arm64: dts: ls1028a: Add ethernet property for
 l2switch CPU port

commit e46ac57e7c1658d957ae248c0d21235c006944dd from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

This enables the CPU traffic for the l2 switch (aka the
CPU frame injection/ extraction feature).

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 2ed5bb158c33..09d00cc014ff 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -852,6 +852,7 @@
                                         };
                                         port@5 {
                                                 reg = <5>;
+						ethernet = <&enetc_port3>;
                                                 fixed-link {
                                                         speed = <1000>;
                                                         full-duplex;
@@ -859,7 +860,7 @@
                                         };
                                 };
                         };
-                        ethernet@0,6 {
+			enetc_port3: ethernet@0,6 {
                                 compatible = "fsl,enetc";
                                 reg = <0x000600 0 0 0 0>;
                                 fixed-link {
-- 
2.17.1

